modules/ROOT/nav.adoc

```diff
@@ -107,6 +107,11 @@
 **** xref:deploy:deployment-option/cloud/managed-connectors/create-mysql-source-connector.adoc[MySQL (Debezium) Source Connector]
 **** xref:deploy:deployment-option/cloud/managed-connectors/create-postgresql-connector.adoc[PostgreSQL (Debezium) Source Connector]
 **** xref:deploy:deployment-option/cloud/managed-connectors/create-snowflake-connector.adoc[Snowflake Sink Connector]
+*** xref:deploy:deployment-option/cloud/api/index.adoc[Cloud API]
+**** xref:deploy:deployment-option/cloud/api/cloud-api-overview.adoc[Cloud API Overview]
+**** xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[Cloud API Authentication]
+**** xref:deploy:deployment-option/cloud/api/cloud-controlplane-api.adoc[Control Plane API]
+**** xref:deploy:deployment-option/cloud/api/cloud-dataplane-api.adoc[Data Plane APIs]
```

Comment:
If these are task topics, they should start with a verb like "Use Control Plane API". Or you could use the actual page title for that one and the data plane one (e.g., Manage Cluster Resources Using the Cloud API").